load-testing/tests/relays_stress_test.go

```diff
@@ -0,0 +1,457 @@
+	sessionInfoNotif
+	newApps      []*accountInfo
+	newGateways  []*accountInfo
+	newSuppliers []*accountInfo
+}
+
+func TestLoadRelays(t *testing.T) {
+	gocuke.NewRunner(t, &relaysSuite{}).Path(filepath.Join(".", "relays_stress.feature")).Run()
+}
+
+func (s *relaysSuite) LocalnetIsRunning() {
+	s.ctx, s.cancelCtx = context.WithCancel(context.Background())
+
+	// Cancel the context if this process is interrupted or exits.
+	// Delete the keyring entries for the application accounts since they are
+	// not persisted across test runs.
+	signals.GoOnExitSignal(func() {
+		fmt.Println("")
+		for _, app := range s.activeApplications {
+			s.txContext.GetKeyring().Delete(app.keyName)
+		}
+		for _, app := range s.preparedApplications {
+			s.txContext.GetKeyring().Delete(app.keyName)
+		}
+		s.cancelCtx()
+	})
+
+	s.Cleanup(func() {
+		for _, app := range s.activeApplications {
+			s.txContext.GetKeyring().Delete(app.keyName)
+		}
+		for _, app := range s.preparedApplications {
+			s.txContext.GetKeyring().Delete(app.keyName)
+		}
+	})
+
+	// Initialize the provisioned gateway and suppliers keyName->URL map that will
+	// be populated from the load test manifest.
+	s.gatewayUrls = make(map[string]string)
+	s.suppliersUrls = make(map[string]string)
+
+	// Set up the blockClient that will be notifying the suite about the committed blocks.
+	s.blockClient = testblock.NewLocalnetClient(s.ctx, s.TestingT.(*testing.T))
+	channel.ForEach(
+		s.ctx,
+		s.blockClient.CommittedBlocksSequence(s.ctx),
+		func(ctx context.Context, block client.Block) {
+			s.latestBlock = block
+		},
+	)
+
+	// Setup the txContext that will be used to send transactions to the network.
+	s.txContext = testtx.NewLocalnetContext(s.TestingT.(*testing.T))
+
+	// Get the relay cost from the tokenomics module.
+	s.relayCost = s.getRelayCost()
+
+	// Setup the tx listener for on-chain events to check and assert on transactions results.
+	s.setupTxEventListeners()
+
+	// Initialize the funding account.
+	s.initFundingAccount(fundingAccountKeyName)
+
+	// Initialize the provisioned gateways and suppliers from the load test manifest.
+	s.initializeProvisionedActors()
+
+	// Some suppliers may already be staked at genesis, ensure that staking during
+	// this test succeeds by increasing the sake amount.
+	minStakeAmount := s.getProvisionedActorsCurrentStakedAmount()
+	stakeAmount = sdk.NewCoin("upokt", math.NewInt(minStakeAmount+1))
+}
+
+func (s *relaysSuite) ARateOfRelayRequestsPerSecondIsSentPerApplication(appRPS string) {
+	relayRatePerApp, err := strconv.ParseInt(appRPS, 10, 32)
+	require.NoError(s, err)
+
+	s.relayRatePerApp = relayRatePerApp
+}
+
+func (s *relaysSuite) TheFollowingInitialActorsAreStaked(table gocuke.DataTable) {
+	// Store the initial counts of the actors to be staked to be used later in the test,
+	// when information about max actors to be staked is available.
+	s.supplierInitialCount = table.Cell(3, 1).Int64()
+	s.gatewayInitialCount = table.Cell(1, 1).Int64()
```

Comment:
`(1,1), (2,1) and (3,1)` make an implicit assumption of how the table in the feature test is structured.

Two asks
1. Is there a way to assert it's correct in the .feature file?
2. Thought on moving something to a constant? E.g. `gatewayRow`, `supplierRow`, etc...
3. My OCD is kicking in with the fact that we're doing `3,1,2` instead of `1,2,3`

Reply:
1. For this table, given that we expect the second column to be an int64, we would expect to see an error if the feature provided a non-integer value.
2. :heavy_check_mark: 
3. :+1: